### DISCUSSION TITLE
part-ii/2018-82/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#335] [Question comments]
Part II 2018 Linear Analysis: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2018-82/

**Comment by stevezshen** [comment]
<details>
<summary>(c)</summary>

Let $F_n = \left\{f\in X^{*}\mid \sum_{k=1}^{\infty}\left|f(x_k)\right|\leq n\right\}$. It is closed, so by Baire's theorem, $F_n$ has non empty interior for some $n$. 

Alternatively, define $T:X^{\ast}\to \ell^{1}$ by $f\mapsto \left(f\left(x_n\right)\right)$. We need to prove that $T$ has closed graph, hence is continuous. Indeed, suppose $f_n\to f$ in $X^{\ast}$, and $\left(f_n \left(x_m\right) \right)_m \to \left(y_m\right)_m$ in $\ell^{1}$, as $n\to\infty$. Then by uniqueness of limits, $f\left(x_m\right) = y_m$. 

</details>
